client/my-sites/customer-home/notices/site-setup-complete-card/index.jsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CelebrateNotice from '../celebrate-notice';
+
+const SiteSetupCompleteCard = ( { displayChecklist } ) => {
```

Comment:
Minor naming note for these. I think we'd probably call these banners or notices vs cards.

Reply:
Agree names are confusing, but decided to stick with a cards terminology for consistency reasons since we're aiming for an API that returns a layout of _cards_. Happy to rename them if rest of folks agree on a better name 🙂